laktory/models/resources/databricks/job.py

```diff
@@ -862,3 +862,8 @@ def terraform_properties(self) -> dict:
             d[k] = _clusters
 
         return d
+
```

Comment:
Can you move this method after the declaration of the job attributes and before the `# Resource Properties` block? This would be line 758.